### PR TITLE
Fix storybook build wireit commands

### PR DESCRIPTION
**Pull request by louwie17**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

**Note:** this will require a rebase after https://github.com/woocommerce/woocommerce/pull/51996 is merged.

This is a follow up of https://github.com/woocommerce/woocommerce/pull/51996 PR and the wireit PR ( https://github.com/woocommerce/woocommerce/pull/51701 ).

In a clean repo the build command for storybook didn't trigger the build commands of the packages, causing it to fail. These changes make sure it runs those commands first.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Either checkout this branch in a fresh clone of the repo or run `git clean -xdf`
2. Run `pnpm i`
3. Run `pnpm --filter=@woocommerce/storybook run build` it should finish successfully
4. Run `npx http-server tools/storybook/storybook-static` and go to the URL
5. It should render the storybook correctly.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

This is a small dev change to fix the storybook Github build process.

</details>

### CODE REVIEW

File: plugins/woocommerce/client/admin/webpack.config.js [hunk 1]
```diff
@@ -212,7 +212,10 @@ const webpackConfig = {
 		new CopyWebpackPlugin( {
 			patterns: [
 				{
-					from: '../../../../packages/js/product-editor/build/blocks',
+					from: path.join(
+						__dirname,
+						'../../../../packages/js/product-editor/build/blocks'
+					),
```
Comment (louwie17) [inline plugins/woocommerce/client/admin/webpack.config.js]: Part of https://github.com/woocommerce/woocommerce/pull/51996

File: tools/storybook/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	"author": "Automattic",
 	"license": "GPL-3.0-or-later",
 	"scripts": {
-		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:storybook$/'",
+		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:.*$/'",
```
Comment (louwie17) [inline tools/storybook/package.json:14]: Run all the `build:project` commands instead of just storybook.

File: tools/storybook/webpack.config.js [hunk 3]
```diff
@@ -104,7 +105,7 @@ module.exports = ( storybookConfig ) => {
 				{
 					from: path.resolve(
 						__dirname,
-						`../../plugins/woocommerce/assets/client/admin/app/*.css`
+						`../../plugins/woocommerce/client/admin/build/app/*.css`
```
Comment (louwie17) [inline tools/storybook/webpack.config.js:108]: Target the `build` css directly, as we have the `admin-library` as a dependency.

**Review by kmanijak — APPROVED** [review]
Works just fine! 👍

**Review by rjchow — APPROVED** [review]
tests well, thanks!

**Pull request merged**